governance-exercises/Le Bateleur - Interpretation Exercise 01/GOV 12 Rewrite.md

```diff
@@ -0,0 +1,56 @@
+# Interpretation Exercise 01
+
+---
+
+## Proposed Rewrite of GOV 12
+
+To redress the aforementioned concerns, I would rewrite GOV 12 to achieve three aims. 
+
+First, I create a general protocol for GOV 12 and restrict its application to "substantial threats" to the functioning of Maker Governance. This is a far higher showing to meet than the current one. It redresses the risk stemming from ambiguity/overbreadth, while preserving to a degree the original "bootstrapping" intention.
+
+Second, I require endorsements from rotating ecosystem actors to provide checks/balances against the concentration of power in Governance Facilitators.
+
+Third, I carve out exceptions to the general protocol for 1) clear mistakes and 2) Launch Project-related needs. The former exclusion for clear mistakes somewhat preserves the original intention behind GOV 12, but in a narrowly tailored way to better guard against misalignment. The latter exclusion is appropriate, I believe, because the [Launch Project](https://mips.makerdao.com/mips/details/MIP108#9-launch-project) is literally framed as an exception to the rule of maximum transparency in governance processes. Further, the single instance of GOV 12 governance practice was the introduction of the EDSR.  In [Discord](https://discord.com/channels/893112320329396265/899667119426371625/1131216377575919716) as well as the [Forum proposal](http://forum.makerdao.com/t/request-for-gov12-1-2-edit-to-the-stability-scope-to-quickly-implement-enhanced-dsr/21405/6?u=le_bateleur), Rune maintained the EDSR was necessary to stoke momentum for the Launch Project (Phase 1). 
+
+My rewrite assumes that associated provisions in the rulebooks (i.e., [Atlas 2.12)](https://mips.makerdao.com/mips/details/MIP101#2-12-scope-bootstrapping-gov12) would have to be interpreted or re-written so as to be reconciled with the revised GOV 12. The revised GOV 12 could complement [MIP24](https://mips.makerdao.com/mips/details/MIP24#sentence-summary), an emergency voting protocol that may be initiated by any community member and manages urgent changes to the protocol. In contrast, the revised GOV 12 is a temporary measure authorizing only Governance Facilitators, under certain circumstances, to expedite revisions to the Atlas and Scopes Artifacts.  
+
+My rewrite also assumes that the Governance Facilitators interpreting GOV 12 are comfortable applying fuzzy legal terms of art. (There would be distinct benefits: the civil law tradition is a rich one going back centuries. Arguably, it is as close as we can get to a stable, predictable foundation for drafting and interpreting governance logic.) 
```

Comment:
Fair enough. I don't really know what any of that means, so I guess if governance facilitators would need to be comfortable with that, some sort of guide or primer on it might be worthwhile alongside the changes.